QuizApp/App/Networking/Models/Category.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+struct Category {
```

Comment:
Why can't you use an enum for this? Also, this should be called CategoryNetworkModel.

Reply:
It can. The Category struct was created to mock the categories. The idea was to refactor it to quiz model, which contains the category enum, in the fetching quiz part.